### ISSUE TITLE
Support PyTorch 0.4 and update FIM

### ISSUE THREAD

**Issue by moskomule** [moskomule/ewc.pytorch#2]
@moskomule

Think https://github.com/moskomule/ewc.pytorch/blob/master/utils.py#L41 is the right way or not.

**Comment by avinashsai** [comment]
@moskomule 
Hi,
Even I am curious to know why the loss is calculated between logits and predicted labels. It should ideally be between logits and true labels.

In utils.py :
Gradients can be retrieved using line no: 44. Why are they retrieved by making predictions again?
